auth-connector/src/generated/main/grpc/com/cloudera/thunderhead/service/authorization/AuthorizationGrpc.java

```diff
@@ -0,0 +1,522 @@
+package com.cloudera.thunderhead.service.authorization;
```

Comment:
why check in the generated file? this is generated no?

Reply:
Yes, it's generated.
At first we didn't want to, but we found no way to generate these files when doing an IDEA Build, which lead to disruption in daily work. Also when switching branches this was annoying, you would had to build the project from command line.

Reply:
https://github.com/google/protobuf-gradle-plugin/issues/111
Tried this plugin, but it was not working for me.